src/universal/modules/meeting/containers/MeetingContainer/MeetingContainer.js

```diff
@@ -12,7 +12,7 @@ import withAtmosphere from 'universal/decorators/withAtmosphere/withAtmosphere';
 import MeetingAgendaFirstCall from 'universal/modules/meeting/components/MeetingAgendaFirstCall/MeetingAgendaFirstCall';
 import MeetingAgendaItems from 'universal/modules/meeting/components/MeetingAgendaItems/MeetingAgendaItems';
 import MeetingAvatarGroup from 'universal/modules/meeting/components/MeetingAvatarGroup/MeetingAvatarGroup';
-import MeetingCheckin from 'universal/modules/meeting/components/MeetingCheckin/MeetingCheckin';
+import MeetingCheckIn from 'universal/modules/meeting/components/MeetingCheckIn/MeetingCheckIn';
```

Comment:
heh, next time you reformat your computer, use a case-sensitive hard drive format. it will save you HOURS of wondering why CI is failing :wink:

Reply:
what the...

Reply:
macOS by default is extended case-insensitive. linux is always case sensitive. when i first pulled this branch it couldn't resolve the file because it asked for it in lower case but the file was in uppercase iirc.